test/tests/functional/pbs_highreslog.py

```diff
@@ -122,22 +145,20 @@ def test_basic_tracejob(self):
         j = Job(TEST_USER)
         jid = self.server.submit(j)
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid)
-        lines = self.server.log_lines(logtype='tracejob', id=jid, n='ALL')
-        if len(lines) > 1:
-            # Remove first line as it is not log line
-            lines = lines[1:]
-        for line in lines:
-            m = self.tm_micro_re.match(line)
-            if m is None:
-                # If this is accounting log, ignore it as
-                # Accounting log does not have high res logging
-                line = line.split()
-                _msg = 'Not Found high resolution time stamp in log'
-                self.assertFalse(len(line) >= 3 and
-                                 (line[2].strip() != 'A'), _msg)
-        _msg = self.server.shortname + \
-            ': High resolution time stamp found in log'
-        self.logger.info(_msg)
+        self.validate_trace_job_lines(jid=jid)
+
+    def test_env_variable_overwrite(self):
+        conf_path = self.du.parse_pbs_config()
+        pbs_init = os.path.join(os.sep, conf_path['PBS_EXEC'],
+                                'libexec', 'pbs_init.d')
+        cmd = ['sudo', 'PBS_LOG_HIGHRES_TIMESTAMP=1', pbs_init, 'restart']
```

Comment:
So, if I'm understanding this test correctly, you are testing that the environment variable will overwrite the pbs.conf variable. In that case, shouldn't you set the pbs.conf variable as well?

Reply:
added this into the test